src/Veldrid/OpenGL/OpenGLFramebuffer.cs

```diff
@@ -77,9 +104,9 @@ public void CreateGLResources()
                         glFramebufferTextureLayer(
                             FramebufferTarget.Framebuffer,
                             GLFramebufferAttachment.ColorAttachment0 + i,
-                            glTex.Texture,
+                            (uint)textureTarget,
                             (int)colorAttachment.MipLevel,
-                            (int)colorAttachment.ArrayLayer);
+                            (int)colorAttachment.ArrayLayer / 6);
```

Comment:
This portion isn't quite correct -- `glFramebufferTextureLayer` expects slightly different parameters and the texture name itself is supposed to be passed here. Also, the array layer shouldn't be divided by 6 -- the face index is included in that layer.